test/testpy/__init__.py

```diff
@@ -61,9 +61,7 @@ def GetCommand(self):
     source = open(self.file).read()
     flags_match = FLAGS_PATTERN.search(source)
     if flags_match:
-      # PORT should match the definition in test/common/index.js.
-      env = { 'PORT': int(os.getenv('NODE_COMMON_PORT', '12346')) }
-      env['PORT'] += self.thread_id * 100
+      env = { }
```

Comment:
Can `env` and the later `.format()` call be removed now, since `env` is now empty? For reference, was introduced in 782620f03ffcaa5be9e84f8f15a638cb7c03e464.

Reply:
@maclover7 Sure seems like it. Will rebase, make that change, test, and force push....